sgx_oc/src/linux/ocall/asyncio.rs

```diff
@@ -60,6 +60,17 @@ pub unsafe fn epoll_create1(flags: c_int) -> OCallResult<c_int> {
     Ok(result)
 }
 
+pub unsafe fn epoll_create(size: c_int) -> OCallResult<c_int> {
+    let mut result: c_int = 0;
+    let mut error: c_int = 0;
+
+    let status = u_epoll_create_ocall(&mut result as *mut c_int, &mut error as *mut c_int, size);
```

Comment:
I think `epoll_create` can be implemented by calling `epoll_create1`  in sgx_oc, which reduces defining `EDL` and u_epoll_create_ocall in sgx_urts.

Reply:
i see, more better solution, thanks for your advice